packages/components/.storybook/addons/manager.ts

```diff
@@ -0,0 +1,26 @@
+import { addons, types } from '@storybook/manager-api';
+import { ADDON_ID_BUG_HINT, TOOL_ID } from './bug-hint/constants';
```

Comment:
Please call both exports just ADDON_ID and change the import via `as` https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import